dataactbroker/fileRoutes.py

```diff
@@ -265,3 +312,23 @@ def wrapped(*args, **kwargs):
                 "submission_id is required", StatusCode.CLIENT_ERROR)
         return fn(submission_id, *args, **kwargs)
     return wrapped
+
+
+def _find_existing_submissions_in_period(sess, submission_id, cgac_code, reporting_fiscal_year,
```

Comment:
Alisa mentioned turning this into a decorator. That'll consolidate the extraction of arguments from request and the check this is doing. So instead a a call in each function, you'll just use `@check_existing_submission_in_period` or something similar. That seems like a good route to take.